services/docker.go

```diff
@@ -43,20 +40,8 @@ func GetContainerInfo(id string) (types.ContainerJSON, error) {
 	return c.ContainerInspect(context.Background(), id)
 }
 
-func GetDaemonInfo(host string) (types.Info, error) {
-	transport := &http.Transport{
-		DialContext: (&net.Dialer{
-			Timeout:   1 * time.Second,
-			KeepAlive: 30 * time.Second,
-		}).DialContext}
-	cli := &http.Client{
-		Transport: transport,
-	}
-	c, err := client.NewClient(host, client.DefaultVersion, cli, nil)
-	if err != nil {
-		return types.Info{}, err
-	}
-	return c.Info(context.Background())
+func GetDaemonInfo(i *Instance) (types.Info, error) {
+	return i.dockerClient.Info(context.Background())
```

Comment:
dockerClient couldn't be `nil` here if it couldn't connect to the daemon?. Shoudln't we check that?